corehq/apps/reports_core/filters.py

```diff
@@ -236,7 +236,7 @@ def value(self, **kwargs):
             choices = selection.split(CHOICE_DELIMITER)
             typed_choices = [transform_from_datatype(self.datatype)(c) for c in choices]
             return [Choice(tc, c) for (tc, c) in zip(typed_choices, choices)]
-        return [Choice(SHOW_ALL_CHOICE, '')]
+        return self.default_value()
 
     def default_value(self):
-        return None
+        return [Choice(SHOW_ALL_CHOICE, '')]
```

Comment:
Hmm, I'm not sure I feel comfortable changing default behavior unless we are confident it won't affect other reports in an adverse way. Is there a way to fix the bug without changing the default behavior, or is there a way we can be sure this fix won't affect current reports?